cypress/support/index.d.ts

```diff
@@ -0,0 +1,5 @@
+declare namespace Cypress {
```

Comment:
Codacy found an issue: ['namespace' and 'module' are disallowed](https://app.codacy.com/gh/Project-Lavinia/Lavinia-client/pullRequest?prid=5953345)